### PR TITLE
VACMS-4918: Updated toolbar permissions for various roles

### LINKED ISSUE #4918: A Content Creator does not have the option to logout
## Background

A user was changed to a content creator role only and now they do not have access to the logout button in the header and cannot log out of the production site. 

Issue discovered by Help Desk while changing editor roles to block them from publishing content before it should be published.  

## Acceptance Criteria

- [ ] All logged in users have access to view the user actions menu in the admin toolbar, including the "Log out" link

### PR DESCRIPTION
**Pull request by netboss**
- Updated various roles permissions in the drupal admin permissions page.
- Updated roles, added permmission in PHPUnit test again.

## Description

closes #4918

## Testing done
Passes all tests

## Screenshots
VACMS-4918-updated-toolbar-permissions

## QA steps

As user administrator
1. [ ] Go to `/admin/people/permissions`, scroll down to "Toolbar" to verify and confirm the "Use toolbar" permission is assigned to the "Authenticated user" (and all other "Content Creator *" roles)

As a user content creator
1. [ ] Verify and confirm that you can see the toolbar

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/user.role.content_creator_benefits_hubs.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ weight: -7
 is_admin: null
 permissions:
   - 'access files overview'
+  - 'access toolbar'
```
Comment (swirtSJW) [inline config/sync/user.role.content_creator_benefits_hubs.yml:16]: Drupal permissions are cumulative.  So  by adding it to role.authenticated, you don't  need to add it to all the others. These other entries can all be removed.

**Review by swirtSJW — CHANGES_REQUESTED** [review]

File: config/sync/user.role.content_creator_benefits_hubs.yml (hunk 1 above)
Comment (netboss) [reply config/sync/user.role.content_creator_benefits_hubs.yml:16]: @swirtSJW 
There were some roles that has the 'access toolbar' permission enabled previously, but now should also be covered by the role.authenticated.

Question: should I remove the 'access toolbar' permission from those too and leave the 'access toolbar' permission enabled only in role.authenticated?

Comment (swirtSJW) [reply config/sync/user.role.content_creator_benefits_hubs.yml:16]: Yes remove all as they become redundant as soon as you add it to all authenticated.

**Review by swirtSJW — APPROVED** [review]
Looks good
image

**Pull request merged**